packages/console/src/monaco/MonacoProviders.tsx

```diff
@@ -296,8 +299,23 @@ class MonacoProviders extends PureComponent<
         };
       });
 
-    const disableActions: monaco.languages.CodeAction[] = diagnostics
+    const seenCodes = new Set<string>();
+    const duplicateCodes = new Set<string>();
+    diagnostics.forEach(d => {
+      if (d.code == null) {
```

Comment:
Should already be filtered out right?

Reply:
Yes, but TS doesn't know that so I had to add this check